sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -500,6 +500,34 @@ def test_experiment_use_multiple_sdk_containers(self):
     self.assertNotIn(
         "use_multiple_sdk_containers", environment.proto.experiments)
 
+  def test_experiment_enable_health_checker(self):
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp',
+         '--experiments', 'beam_fn_api'])
+    environment = apiclient.Environment(
+        [], pipeline_options, 1, FAKE_PIPELINE_URL)
+    self.assertIn("enable_health_checker", environment.proto.experiments)
+
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp',
+         '--experiments', 'beam_fn_api',
+         '--experiments', 'enable_health_checker'])
+    environment = apiclient.Environment(
+        [], pipeline_options, 1, FAKE_PIPELINE_URL)
+    self.assertIn("enable_health_checker", environment.proto.experiments)
+
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp',
+         '--experiments', 'beam_fn_api',
+         '--experiments', 'disable_health_checker'])
+    environment = apiclient.Environment(
+        [], pipeline_options, 1, FAKE_PIPELINE_URL)
+    self.assertNotIn(
+        "enable_health_checker", environment.proto.experiments)
```

Comment:
We can also add 
`    self.assertIn('disable_health_checker', environment.proto.experiments)`

Reply:
Done.